types/knockout/index.d.ts

```diff
@@ -567,7 +567,7 @@ interface KnockoutStatic {
 
         readValue(element: HTMLElement): any;
 
-        writeValue(element: HTMLElement, value: any): void;
+        writeValue(element: HTMLElement, value: any,  allowUnset: boolean): void;
```

Comment:
- I think it should be optional, to not introduce breaking change; according to code it doesn't rely on any particular value to be certainly there
- please fix double space